txwinrm/enumerate.py

```
@@ -241,13 +241,16 @@ def connectionLost(self, reason):
             #  Decrypt data first
             data = self._sender.decrypt_body(''.join(self._data))
             self._debug_data = data
-            self._xml_parser.feed(data)
+            try:
+                self._xml_parser.feed(data)
+            except Exception:
+                log.debug('Could not parse data {}'.format(data))
```

It seems like this is a fatal condition of the enumeration operation worthy of more than a debug log. I feel like this would result in silent failure.